docs/computer/Acoustics.md

```diff
@@ -29,8 +29,8 @@ Points would be awarded as follows. If the AUV successfully causes the ball to d
 flare with the pinger: 40 Points
 [This task from the [rulebook of SAUVC 2018](https://sauvc.org/2018/)]
 
-![yellow_Flare_animated](https://github.com/auvzhcet/Documentation/tree/jawad-patch-1/docs/computer/static/flare_Animation.png)
-![yellow_Flare_underwater](https://github.com/auvzhcet/Documentation/tree/jawad-patch-1/docs/computer/static/yellow_Flare_water.png)
+![yellow_Flare_animated](https://github.com/auvzhcet/Documentation/tree/master/docs/computer/static/flare_Animation.png)
```

Comment:
This is not the correct way to add images. You are actually pointing GitHub link. Do it like this:
![Flare animation](static/flare_Animation.png)